<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>634</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Зашифровать</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Расшифровать</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="InputColomsKeys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelColomsKeys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ключи для столбцов</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите  текст для зашифровки</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>711</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="OutputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>711</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelOutputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифрованный текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="AttentionLabelInputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечания:</string>
    </property>
   </widget>
   <widget class="QLabel" name="AttentionLabelOutputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечания:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>выполнить зашифровку</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="AttentionLabelInputColomsKeys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечания:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AttentionTextColomsKeys">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.ключи должны быть числовыми значениями из множества {1,2,3,4}.
2.требуется ввести 4 ключа.
3.ключи должны вводиться через пробел.</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AttentionTextInputText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.текст не должен содержать символа &quot;|&quot;, так как он зарезервирован программой в качестве разделителя при выводе блоков зашифрованного текста. </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AttentionTextOutputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.будет выведено 12 блоков рассечённых-разнесённых символов.
2.блоки выводятся последовательно(от 1-ого до 12-ого).
3.в качестве разделителя блоков исползуется символ &quot;|&quot;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InputRowsKeys">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRowsKeys">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ключи для строк</string>
    </property>
   </widget>
   <widget class="QLabel" name="AttentionLabelInputRowsKeys">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечания:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AttentionTextRowsKeys">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.ключи должны быть числовыми значениями из множества {1,2,3}.
2.требуется ввести 3 ключа.
3.ключи должны вводиться через пробел.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
